cloudinstall/utils.py

```diff
@@ -771,6 +771,26 @@ def download_url(url, output_file):
             url, res.content))
 
 
+def pollinate(session, tag):
+    """ fetches random seed
+
+    :param str session: randomly generated session id
+    :param str tag: custom tag
+    """
+    if not os.path.isfile('/usr/bin/pollinate'):
+        return
+    if tag not in ['IL', 'IS', 'IM', 'DL', 'DM', 'DS']:
+        raise UtilsException("Unknown TAG {}".format(tag))
+    agent_str = 'uoi/{}-{}'.format(session, tag)
+    try:
+        check_call(['pollinate', '-q',
+                    '--curl-opts',
+                    '--user-agent',
+                    agent_str])
```

Comment:
need to send the all the curl opts as one option. if you ran this on the command-line, you'd get an error from pollinate: 
```
 sudo -u pollinate pollinate -r  --curl-opts --user-agent FOO
Nov 19 18:54:07 responsible-celery pollinate[5581]: ERROR: Unknown options [FOO]
```
It needs to be like this:
```python
check_call(['pollinate', '-q', '--curl-opts', '--user-agent {}'.format(agent_str)])
```